gsoc/index.md

```diff
@@ -8,14 +8,16 @@ permalink: /gsoc/
  
 [Background on GSoC: start here!](./background.html)
 
-## GSoC/Open Astronomy 2018
+## GSoC/Open Astronomy 2019
 
 OpenAstronomy is an umbrella organisation which collects project ideas
-from any of its members. OpenAstronomy was a GSOC mentoring organisation in 2016 and 2017 and is participating again
-in 2018. 
+from any of its members. OpenAstronomy was a GSOC mentoring organisation since 2016 and
+it's applying to participate
+in 2019.
 
 Student applications to OpenAstronomy projects follows the same
-rules as the [Python Software Foundation] and the [GSoC Student Guide], our guide to a good application is the [OpenAstronomy Student Guide].
+rules as the [Python Software Foundation] and the [GSoC Student Guide],
+we have [also our own guide to a good application for OpenAstronomy][OpenAstronomy Student Guide].
```

Comment:
does this render this way?

Reply:
It should... or so it says in this [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)

Reply:
No... it seems it's not rendering :(
[result](https://70-30926520-gh.circle-artifacts.com/0/site/gsoc/index.html)

Reply:
Ah no, it does :)

[result](https://71-30926520-gh.circle-artifacts.com/0/site/gsoc/index.html) after adding some blank spaces...